src/main/java/org/example/hw12/JewelFinder.java

```diff
@@ -0,0 +1,19 @@
+package org.example.hw12;
+
+public class JewelFinder {
+
+    public int numJewelsInStones(String jewels, String stones) {
+        char[] jewelsSinglyUniq = jewels.toCharArray();
+        char[] stonesSingly = stones.toCharArray();
+        int counter = 0;
+
+        for (char j : jewelsSinglyUniq) {
+            for (char s : stonesSingly) {
```

Comment:
bad var name

Reply:
fixed, i used Set before, var name forgot to change =/